File: app/src/main/java/com/nuhkoca/myapplication/data/remote/video/Pictures.java
package com.nuhkoca.myapplication.data.remote.video;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.nuhkoca.myapplication.BR;

import java.util.List;

import androidx.databinding.BaseObservable;
import androidx.databinding.Bindable;

public class Pictures extends BaseObservable {

    @Expose
    @SerializedName("sizes")
    private List<Size> sizes;

    @Bindable
    public List<Size> getSizes() {
        return sizes;
    }

    public void setSizes(List<Size> sizes) {
        this.sizes = sizes;
        notifyPropertyChanged(BR.sizes);
    }
}

File: app/src/main/java/com/nuhkoca/myapplication/data/remote/video/Pictures.kt
package com.nuhkoca.myapplication.data.remote.video

import androidx.databinding.BaseObservable
import androidx.databinding.Bindable
import com.google.gson.annotations.SerializedName
import com.nuhkoca.myapplication.BR

data class Pictures(@SerializedName("sizes") var _sizes: List<Size>) : BaseObservable() {

    var sizes: List<Size>
        @Bindable get() = _sizes
        set(value) {
            _sizes = value
            notifyPropertyChanged(BR.sizes)
        }
}
